util/storeutil/storeVars.go

```diff
@@ -0,0 +1,8 @@
+package storeutil
+
+import (
+	atomic2 "go.uber.org/atomic"
```

Comment:
Why use an alias (there is no package name conflict)?

Reply:
Addressed.